### DISCUSSION TITLE
Certificados de participacion

### DISCUSSION THREAD

**Discussion by JuanSposada** [Launch-X-Latam/MisionFrontEnd#599] [Team Posts]
Compañeros, a alguno de ustedes ya les llegó el certificado de participación? Llene un formulario en diciembre pero aun no me llega nada, saludos

**Comment by DavidSantana872** [comment]
Cual formulario? A mi nunca me llegó ningún correo con ningún formulario 😶😶😶😶

**Comment by Max80318** [comment]
Ese formulario era para que microsoft nos recomendara a algunas empresas
para trabajo eso entendi yo

**Comment by Amhedriel** [comment]
## Buenas!

El formulario que llenamos antes del 29 de Diciembre es como dice Max para que LaunchX Latam pueda enviarlo a empresas interesadas en contratar. Pero eso también depende de ciertas cosas como tener nuestras prácticas terminadas, haber tenido participaciones en las clases y acá en GitHub, el haber ayudado a los compañeros, y muchos otros factores que fueron explicando en el transcurso del programa.

### **Certificado**
Con respecto al certificado, hasta donde tengo entendido se nos entregará algo así como una constancia de participación una vez terminado el **"Hackatton"** a fines de Enero, no se en que grado el no haber enviado el formulario afecte a este certificado de **constancia** porque literalmente estuvo publicado 15 días antes.

Bien, cualquier información que tengan compártanla, abrazos.

**Comment by JolisteR** [comment]
alguien me dice donde esta formulario no lo e llenado

**Comment by Amhedriel** [comment]
> alguien me dice donde esta formulario no lo e llenado

El formulario que llenamos antes del 29 de Diciembre, ya no está disponible, por eso debíamos estar participando activamente, incluso se dio recomendaciones en los directos, lo siento por los que no pudieron llegar a llenarlo, por si acaso pregunten a contacto@innovaccion.mx, saludos

**Comment by floressdi** [comment]
No pude llenarlo :c

**Comment by AnaMejiaR** [comment]
:( Aunque igual siempre pienso que lo más importante es lo que aprendes, ponerlo en práctica para no olvidarlo y mejorar continuamente.

**Comment by luiserdef** [comment]
> :( Aunque igual siempre pienso que lo más importante es lo que aprendes, ponerlo en práctica para no olvidarlo y mejorar continuamente.

Claro, lo importante es lo que se aprende.

**Comment by lODIN007l** [comment]
pero les ha llegao ?

**Comment by Amhedriel** [comment]
no, aún nada

**Comment by Jamesteven** [comment]
Aún nada no han dicho nada si saben algo avisen

**Comment by demariamed** [comment]
Info exclusiva:

Habra un Hack de cierre y se les darán las fechas de cierre personalmente. Sean pacientes.

Saludos

**Comment by Sebastian-Longhitano** [comment]
¿Y respecto de la pregunta de Juan sobre los certificados de participación se sabe algo al respecto?

Gracias!

**Comment by RIvanCF** [comment]
Ya no les creo

**Comment by MrAlizarinCrimson** [comment]
Bueno al menos aprendieron a programar equisde

**Comment by Talk1504** [comment]
Hola alguien sabe algo sobre la constancia?
________________________________
De: Juan Sebastian Moreno Posada ***@***.***>
Enviado: sábado, 14 de enero de 2023 23:14
Cc: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-backend] Certificados de participacion (#140)


Compañeros, a alguno de ustedes ya les llegó el certificado de participación? Llene un formulario en diciembre pero aun no me llega nada, saludos

**Reply by demariamed** [reply]
A un amigo ya le llego.

**Comment by demariamed** [comment]
Buenas tardes. ¿A alguno de ustedes ya les llegó su certificado?

**Comment by Sebastian-Longhitano** [comment]
No, y dudo que llegue... dile a tu amigo si puede por favor que lo adjunte para ver como es. Gracias!!

**Reply by demariamed** [reply]
Tiene que llegarles como "Innovación Virtual" o "Constancia Innovación Virtual"

**Reply by Mor4n** [reply]
Un chico en LinkedIn llegó a subir la constancia que le dieron
[Anexo link](https://www.linkedin.com/posts/ewrc01_microsoft-frontend-backend-activity-7051415787990839296-2esx?utm_source=share&utm_medium=member_desktop)

¿A alguien más le ha llegado?
De mi parte mandé un correo justo hoy, aunque creo que ya es un poco tarde hahaha...

**Reply by Sebastian-Longhitano** [reply]
Buenas como estás? yo me había resignado... veo que a algunos les llegó y a otros no...

**Reply by Mor4n** [reply]
Holahola, mandé correo justo hoy, ojalá nos lo dieran.
Otra cosa: En el teams dijeron que le preguntaron al chico del linkedin como le hizo y comentó esto:

> - Recibí el certificado al registrarme en un enlace que ellos colocaron en teams hace meses.
> - Y hasta el día de ayer recibí el certificado, desconozco si hay algo más, eso sí, fui de los primeros en registrarme es probable que sea por eso
> 
> - Imagino que los que se registraron después, se los irán entregando en los próximos días

Esto fue enviado por un chico en el teams de microsoft en el canal de chismesito el 12 de abril.

¿Hay alguien aquí que le haya mandado correo al Contacto de innovaccion? si es así ¿Les dieron alguna respuesta o algo? n_n hay que mantenernos informados

**Comment by Sebastian-Longhitano** [comment]
Gracias por responder!
No, a mi no me ha llegado absolutamente nada... no creo que me llegue nada, lamentablemente... dado que es importante tener la certificación... pero bueno, me quedaré con lo aprendido.

Un saludo!

**Reply by demariamed** [reply]
Disculpa a que te refieres con tener la certificación? Aquí andamos 😉

**Reply by Sebastian-Longhitano** [reply]
A que te den un papel (o bueno, en este caso sería digital) en donde conste que participaste del curso de innovación virtual, y que acredite el tiempo de duración, y los temas vistos... (bah normalmente tiene eso, pero en si, la confección depende de quien lo emita)
